dask/array/core.py

```diff
@@ -1181,19 +1181,33 @@ def to_dask_dataframe(self, columns=None):
         from ..dataframe import from_dask_array
         return from_dask_array(self, columns=columns)
 
-    def __int__(self):
-        return int(self.compute())
-
     def __bool__(self):
-        return bool(self.compute())
+        if self.size > 1:
+            raise ValueError("The truth value of a {0} is ambiguous. "
+                             "Use a.empty, a.bool(), a.item(), a.any() or a.all()."
```

Comment:
Just checked and didn't find `empty` or `bool` for NumPy Arrays. `item` exists for NumPy Arrays, but not Dask Arrays. FWICT it would give a similar error. The other two seem fine though.

Reply:
Thanks!

Reply:
np. FWIW added a note about adding `item` for Dask Arrays in the future in issue ( https://github.com/dask/dask/issues/2959 ).